src/Auth0.OidcClient.Android/Resources/Resource.Designer.cs

```diff
@@ -27,6 +27,33 @@ static Resource()
 		public partial class Attribute
 		{
 			
+			// aapt resource value: 0x7f010007
+			public static int font = 2130771975;
```

Comment:
🙀If this is generated by you, as the file you removed says, use `R` references instead of doing that. The next release may change these values and break your build. If it's auto-generated, ignore this comment.

Reply:
I didn't add this - all the `.Designer.` files in .NET are generated/controlled by the tooling.